app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -18,17 +18,23 @@ import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
 import androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
 import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withParent
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.google.common.truth.Truth.assertThat
 import dagger.Component
+import javax.inject.Inject
+import javax.inject.Singleton
```

Comment:
This is causing the ktlint failure, also I think this change is not required in this PR as it is just a replacing of line. Could you try fixing this and push again.

Reply:
Removing these causing errors

Reply:
@viktoriias Actually @anandwana001 is mentioning that this PR changes the order of the imports. We do need all these imports. But the order is incorrect. All `javax` related imports should come in last and in alphabetical order.

Reply:
Done thanks, I moved `javax` down. Although, Android studio keeps trying to move it back because `javax` comes before `org` alphabeticaly.